store/tikv/snapshot.go

```diff
@@ -407,13 +401,7 @@ func (s *tikvSnapshot) get(ctx context.Context, bo *Backoffer, k kv.Key) ([]byte
 		}
 	})
 
-	cli := clientHelper{
-		LockResolver:      s.store.lockResolver,
-		RegionCache:       s.store.regionCache,
-		minCommitTSPushed: &s.minCommitTSPushed,
-		Client:            s.store.client,
-		resolveLite:       true,
```

Comment:
is it as desired to change `resolveLite` be `true` after this change ~?

Reply:
No. It should be a mistake :(